app/views/tweets/tweets.erb

```diff
@@ -0,0 +1,13 @@
+<h2>Welcome, <%= @user.username %>!</h2>
+<br />
+<% User.all.each do |use| %>
```

Comment:
Convention would be to make a call to the database in the controller and save that data in an instance variable. You would then use that instance variable in the view. 

`<% @users.each do |use| %>`